docs/content/functioneel/zaaktypen_api.md

```diff
@@ -0,0 +1,199 @@
+| *Verwijderen*  | Nog niet beschikbaar.     |
+| - voorwaarde   | Nog te bepalen.  |
+| - gevolg       | Nog te bepalen.             |
+| *Raadplegen lijst*   |      |
+| - voorwaarde   | Om roltypeinformatie op te kunnen vragen moet de consumer weten in welke catalogus en bij welk zaaktype het roltype is opgenomen. |
+| - gevolg       | Een lijst met roltypeinformatie behorend bij het zaaktype wordt teruggegeven aan de consumer.             |
+| *Raadplegen details*   |      |
+| - voorwaarde   | Om roltypeinformatie op te kunnen vragen moet de consumer weten in welke catalogus en bij welk zaaktype het roltype is opgenomen. Tevens moet de identficatie van het roltype wat uitgevraagd wordt bekend zijn. |
+| - gevolg       | Roltypeinformatie wordt teruggegeven aan de consumer.             |
+| Gegevens       | Objecttype ROLTYPE <br>Zie de volgende tabel voor de gegevens van deze resource                                 |
+| Specificaties  | https://ref.tst.vng.cloud/ztc/api/v1/schema/#operation/roltype_read                                                                                                   |
+| Samenhang      | - |
+| Bijzonderheden | - |
+
+       
+### Resource: Statustypen
+
+| **Aspect**     | **Beschrijving**                                                                                                                                                                 |
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Aanleiding     | Een consumer wil statustypeinformatie van een ZAAKTYPE zoals opgenomen in de CATALOGUS van de organisatie opvragen.           |
+| *Toevoegen*    | Nog niet beschikbaar.     |
+| - voorwaarde   | Nog te bepalen.  |
+| - gevolg       | Nog te bepalen.             |
+| *Wijzigen*     | Nog niet beschikbaar.     |
+| - voorwaarde   | Nog te bepalen.  |
+| - gevolg       | Nog te bepalen.             |
+| *Verwijderen*  | Nog niet beschikbaar.     |
+| - voorwaarde   | Nog te bepalen.  |
+| - gevolg       | Nog te bepalen.             |
+| *Raadplegen lijst*   |      |
+| - voorwaarde   | Om statustypeinformatie op te kunnen vragen moet de consumer weten in welke catalogus en bij welk zaaktype het statustype is opgenomen. |
+| - gevolg       | Een lijst met statustypeinformatie wordt teruggegeven aan de consumer.             |
+| *Raadplegen details*   |      |
+| - voorwaarde   | Om statustypeinformatie op te kunnen vragen moet de consumer weten in welke catalogus en bij welk zaaktype het statustype is opgenomen. Tevens moet de identficatie van het statustype wat uitgevraagd wordt bekend zijn. |
+| - gevolg       | Statustypeinformatie wordt teruggegeven aan de consumer.             |
+| Gegevens       | Objecttype STATUSTYPE <br>Zie de volgende tabel voor de gegevens van deze resource                                 |
```

Comment:
Geen relatie naar het Zaaktype waar het bijhoort?

Reply:
Op dit moment staat die relatie niet opgenomen in de API dus ook niet beschreven. Zie ook de uitleg van Sergei.